bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -28,6 +28,18 @@
 
 	</thing-type>
 
+	<thing-type uid="plugwiseha:zone">
+
+		<instruction-set targetVersion="1">
+			<update-channel id="temperature">
+				<type>system.indoor-temperature</type>
```

Comment:
```suggestion
				<type>system:indoor-temperature</type>
```

Did it work with `.` as well?

Please check others.

Reply:
Not sure why it didnt show up in my first tests. Re-tested and fixed.